deltachat-ffi/deltachat.h

```diff
@@ -1739,8 +1739,8 @@ dc_contact_t*   dc_get_contact               (dc_context_t* context, uint32_t co
  * - **DC_IMEX_EXPORT_BACKUP** (11) - Export a backup to the directory given as `param1`.
  *   The backup contains all contacts, chats, images and other data and device independent settings.
  *   The backup does not contain device dependent settings as ringtones or LED notification settings.
- *   The name of the backup is typically `delta-chat.<day>.bak`, if more than one backup is create on a day,
- *   the format is `delta-chat.<day>-<number>.bak`
+ *   The name of the backup is typically `delta-chat.<day>.tar`, if more than one backup is create on a day,
```

Comment:
the exported file name is `delta-chat-backup-<day>.tar` (dash before date) which is totally fine, it is just the documentation that's wrong here (and was wrong before)

Reply:
Done.